app/js/main.js

```diff
@@ -18,6 +19,7 @@ angular.element(document).ready(function() {
   angular.module('theshelf',
     [
       'theshelf.core',
+      'theshelf.api_mocks',
```

Comment:
@ronaldofs:
> you need this mocks module to be loaded on the app or is it possible to only inject it during tests?

@azevedo-252:
Unfortunately, from what I've researched and experimented, yes it does.
That's why I did that little hack with browserify to ignore this module's files when we're in development. Because, by injecting ngMockE2E all requests are automatically stubbed.